src/backend/distributed/utils/maintenanced.c

```diff
@@ -271,6 +276,17 @@ CitusMaintenanceDaemonMain(Datum main_arg)
 		int latchFlags = WL_LATCH_SET | WL_TIMEOUT | WL_POSTMASTER_DEATH;
 		double timeout = 10000.0; /* use this if the deadlock detection is disabled */
 		bool foundDeadlock = false;
+		/* collect statistics every 24 hours */
+		const double statistics_collection_interval = 24.0 * 3600.0;
+		/*
+		 * ... but don't collect in first 30 minutes, to not report short living
+		 * instances like in CI tests.
+		 */
+		const double statistics_collection_start = 30.0 * 60.0;
```

Comment:
This seems like a bit of a heuristic. Are we sure we want to keep it? I thought we discussed not doing this.

If we do it, I'd prefer `statsCollectionQuietPeriod`, since `start` can be confusing in time-related code.